test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_DeleteConsumerGroupOffsets.cs

```
@@ -0,0 +1,120 @@
+
+                var committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                List<TopicPartition> topicPartitionToReset = new List<TopicPartition>() { new TopicPartition(topic1.Name, 0) };
+                var res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId1, topicPartitionToReset).Result;
+                Assert.Equal(1, res.Count);
+                Assert.Equal(groupId1, res[0].Group);
+                Assert.Equal(1, res[0].Partitions.Count);
+                Assert.Equal(0, res[0].Partitions[0].Partition.Value);
+                Assert.Equal(ErrorCode.NoError, res[0].Partitions[0].Error.Code);
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(Offset.Unset, committedOffsets[0].Offset);
+
+                // Consumer is actively subscribed to the topic
+                assignmentDone = false;
+                consumer1.Subscribe(new List<String>() { topic1.Name });
+                while (!assignmentDone)
+                {
+                    // To handle the rebalance
+                    consumer1.Consume(TimeSpan.FromSeconds(1));
+                }
+                consumer1.Commit(new List<TopicPartitionOffset>() { new TopicPartitionOffset(topic1.Name, 0, offsetToCommit) }); //commit some offset for consumer
+
+                committedOffsets = consumer1.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                topicPartitionToReset = new List<TopicPartition>() { new TopicPartition(topic1.Name, 0) };
+                res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId1, topicPartitionToReset).Result;
```

i think this should throw an exception.

---

Throwing an exception now, updated the test